src/components/widgets/SearchBar.tsx

```diff
@@ -26,7 +26,7 @@ export class SearchBar extends React.Component {
           </span>
         </div>
         <div className='control'>
-          <Link to={`/search/${this.state.value}`} className='button is-danger'>
+          <Link to={`/search/${this.state.value}`} className='button'>
```

Comment:
hmmm

Reply:
this was just because I hadn't taken it out in the previous style removal task